src/main/java/frc/robot/subsystems/swerve/odometryThread/OdometryThread.java

```diff
@@ -47,7 +48,7 @@ static Queue<Angle> registerInput(Supplier<Angle> supplier) {
   }
 
   static OdometryThread createInstance(DeviceCANBus canBus) {
-    return switch (Robot.CURRENT_ROBOT_MODE) {
+    return switch (Constants.currentMode) {
```

Comment:
What is this? Why do we just have a constants class with stuff inside it and why is current mode not SCREAMING_SNAKE_CASE?

Reply:
#82